src/components/Homepage/subcomponents/BannerSection.tsx

```diff
@@ -12,9 +13,11 @@ const BannerSection = () => {
       <span>{BANNER_SPAN}</span>
       <h1>{BANNER_TITLE}</h1>
       <p>{BANNER_DESC}</p>
-      <button type="button" aria-label={`go to ${BANNER_TITLE} product page`}>
-        SEE PRODUCT
-      </button>
+      <Link href={`/headphones/${BANNER_TITLE}`}>
+        <button type="button" aria-label={`go to ${BANNER_TITLE} product page`}>
```

Comment:
do we need button here? since we're already using link

Reply:
Moved the buttons to a separate client component <RedirectButton /> for all instances.